src/css3/walker.rs

```diff
@@ -76,8 +76,8 @@ fn inner_walk(node: &Node, depth: usize, f: &mut dyn Write) -> Result<(), std::i
             }
         }
         NodeType::Comment { .. } => {}
-        NodeType::Cdo => {}
-        NodeType::Cdc => {}
+        // NodeType::Cdo => {}
+        // NodeType::Cdc => {}
```

Comment:
Dead/Duplicated code. You commented these lines out, just to add them below (L270-271)